roles/ovirt-infra/meta/main.yml

```diff
@@ -26,3 +26,4 @@ dependencies:
   - ovirt-storages
   - ovirt-aaa-jdbc
   - ovirt-permissions
+  - ovirt-external-providers
```

Comment:
Maybe we should call it before ovirt-permission, so maybe someone want to setup permissions for the external provider...